src/types/FilterBySelect.ts

```diff
@@ -0,0 +1,5 @@
+export enum FilterBySelect {
+  All = 'all',
+  Active = 'active',
+  Completed = 'completed',
```

Comment:
enums keys wroten uppercased
```suggestion
  ACTIVE = 'active',
  COMPLETED = 'completed',
```